requirements.txt

```diff
@@ -1,2 +1,2 @@
-pelican==3.5.0
+pelican==3.7.1
```

Comment:
Is this intentional?

Reply:
Yes, it is required to start the devserver smoothly, specially to satisfy the dependency with codehighlight for markdown.